packages/form-js-viewer/src/render/hooks/useKeyDownAction.js

```diff
@@ -0,0 +1,18 @@
+import { useEffect } from 'preact/hooks';
+
+export default function useKeyDownAction(targetKey, action, listenerElement = window) {
```

Comment:
If possible, let's have unit tests for this, as it is heavily used in the component.

Example: https://github.com/bpmn-io/properties-panel/blob/main/test/spec/hooks/useStickyIntersectionObserver.spec.js